temporalio/worker/_workflow_instance.py

```diff
@@ -790,26 +831,33 @@ def _apply_resolve_child_workflow_execution(
             raise RuntimeError(
                 f"Failed finding child workflow handle for sequence {job.seq}"
             )
+        payload_converter, failure_converter = self._converters_with_context(
```

Comment:
Arguably you can store these converters on the child/activity handle instead of recreating the same thing for output that you already created for input, but not that important